API.pm

```diff
@@ -103,6 +103,11 @@ sub getImageUrl {
 		my $image = $images->{L} || $images->{M} || $images->{S};
 		$data->{cover} = $image->{url} if $image;
 	}
+	elsif (my $images = $data->{images}) {
+		# I'll let you decided, but LARGE is 1500x1500 and that's A LOT of data in the database
```

Comment:
That's an odd one: in the `mixImages` case above, used for the "My Mixes" menu in the main menu, `L` and `M` would be the same (640px), here, the same menu, but returned form the `page` endpoint, `LARGE` would be really large, while `M` would be the same 640px. So maybe change the order to `MEDIUM` -> `SMALL` -> `LARGE` in this case?